solstice/src/main/java/dev/equo/ide/BuildPluginIdeMain.java

```diff
@@ -120,6 +135,8 @@ public void launch() throws IOException, InterruptedException {
 			}
 			vmArgs.add("-Dorg.slf4j.simpleLogger.defaultLogLevel=" + (isBlocking ? "info" : "error"));
 
+			vmArgs.add("-Dchromium.args=--disable-web-security");
```

Comment:
I'm guessing this is needed for the Eclipse help system? We should only enable this if in equo mode, and if there's a way we can limit it somehow to be more secure that would be very good as well.

Reply:
Can we remove this flag? Is Eclipse Help the driver for this flag?

Reply:
This flag is required for Chromium to be able to access the filesystem, also, in some cases for BrowserFunctions to work.

[https://docs.equoplatform.com/chromium/106.x/troubleshooting/common-issues.html# _browserfunctions_are_not_executed_in_local_files](url)

Reply:
In https://github.com/equodev/equo-ide/pull/123/commits/0dfd3498f34a31a7010289742eb4bc022d55c918 I removed the flag by default and added docs to the README explaining how and when to add the flag (vmArgs is not actually implemented yet). Does that look good to you? Or do you think it needs to be the default?